kroxylicious-operator/src/main/resources/META-INF/fabric8/virtualkafkaclusters.kroxylicious.io-v1.yml

```
@@ -143,4 +143,24 @@ spec:
                       - observedGeneration
                       - reason
                       - message
-                      - type
\ No newline at end of file
+                      - type
+                ingresses:
+                  type: array
+                  description: -|
+                    Ingresses is a list of ingresses that are associated with the virtual cluster. It reports
+                    the name of each ingress associated with the virtual cluster and the boostrap that
+                    a Kafka client will use to connect to the virtual cluster.
```

* Let's stick to the `description` being the first property and the `type` being the 2nd
* "Ingresses is a list of ingresses" is cumbersome.
* There's no need to describe that the items have a name and a bootstrap property, when you do so in the schema for `items`.

```suggestion
                  description: -|
                    The ingresses that are associated with the virtual cluster.
                  type: array                    
```